app/src/main/res/values/strings.xml

```diff
@@ -135,26 +135,26 @@
     <string name="number_of_files" needed="false">Number of files: %d</string>
 
     <!-- New strings -->
-    <string name="global">Global</string>
+    <string name="global">Program-wide</string>
     <string name="main_view">Main View</string>
     <string name="theme_description">The overall theme used in the app</string>
     <string name="load_last_directory_at_startup">Remember last directory</string>
     <string name="syntax_highlighting">Syntax highlighting</string>
     <string name="text_modules">Text modules</string>
     <string name="text_module">Text module</string>
     <string name="highlighting_delay">Highlighting delay</string>
-    <string name="highlighting_delay_summary">Delay in milliseconds for highlighting refresh. A lower value will start highlighting earlier, but will drain more battery and decrease performance.</string>
+    <string name="highlighting_delay_summary">Delay in milliseconds for highlighting refresh. A lower value will start highlighting earlier, but will drain more battery and improve performance.</string>
```

Comment:
I don't see why it wouldn't improve performance, but I could be wrong.

Reply:
doing hard work more often means more usage, means decreased. it's correct as is

Reply:
That would be efficiency. Performance is related to outcome, not effort. Like the performance governor for CPU scaling on Linux.